[2022-09-26T11:18:03.635+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_dag.model.migrate_to_dbt.cast_obj scheduled__2021-10-16T00:00:00+00:00 [queued]>
[2022-09-26T11:18:03.657+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_dag.model.migrate_to_dbt.cast_obj scheduled__2021-10-16T00:00:00+00:00 [queued]>
[2022-09-26T11:18:03.657+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-09-26T11:18:03.658+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2022-09-26T11:18:03.658+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-09-26T11:18:03.698+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): model.migrate_to_dbt.cast_obj> on 2021-10-16 00:00:00+00:00
[2022-09-26T11:18:03.706+0000] {standard_task_runner.py:54} INFO - Started process 1811 to run task
[2022-09-26T11:18:03.712+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dbt_dag', 'model.migrate_to_dbt.cast_obj', 'scheduled__2021-10-16T00:00:00+00:00', '--job-id', '1196', '--raw', '--subdir', 'DAGS_FOLDER/dbt_dag.py', '--cfg-path', '/tmp/tmplq8c5wub']
[2022-09-26T11:18:03.713+0000] {standard_task_runner.py:83} INFO - Job 1196: Subtask model.migrate_to_dbt.cast_obj
[2022-09-26T11:18:03.715+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/dbt_dag.py
[2022-09-26T11:18:03.855+0000] {task_command.py:384} INFO - Running <TaskInstance: dbt_dag.model.migrate_to_dbt.cast_obj scheduled__2021-10-16T00:00:00+00:00 [running]> on host 3c654011b3f0
[2022-09-26T11:18:04.020+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=isaaclucky88@gmail.com
AIRFLOW_CTX_DAG_OWNER=astronomer
AIRFLOW_CTX_DAG_ID=dbt_dag
AIRFLOW_CTX_TASK_ID=model.migrate_to_dbt.cast_obj
AIRFLOW_CTX_EXECUTION_DATE=2021-10-16T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-16T00:00:00+00:00
[2022-09-26T11:18:04.023+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-09-26T11:18:04.024+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '\n            cd /opt/***/migrate_to_dbt &&\n            dbt --no-write-json run --target prod --models cast_obj\n            ']
[2022-09-26T11:18:04.040+0000] {subprocess.py:86} INFO - Output:
[2022-09-26T11:18:04.043+0000] {subprocess.py:93} INFO - /bin/bash: line 3: dbt: command not found
[2022-09-26T11:18:04.044+0000] {subprocess.py:97} INFO - Command exited with return code 127
[2022-09-26T11:18:04.063+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 127.
[2022-09-26T11:18:04.068+0000] {taskinstance.py:1406} INFO - Marking task as UP_FOR_RETRY. dag_id=dbt_dag, task_id=model.migrate_to_dbt.cast_obj, execution_date=20211016T000000, start_date=20220926T111803, end_date=20220926T111804
[2022-09-26T11:18:04.089+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 1196 for task model.migrate_to_dbt.cast_obj (Bash command failed. The command returned a non-zero exit code 127.; 1811)
[2022-09-26T11:18:04.125+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2022-09-26T11:18:04.249+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
